### PR TITLE
Improvement translation for Spanish language

### PR DESCRIPTION
**Pull request by roliverosc-zz**
Regards @stefansielaff

### CODE REVIEW

File: resources/language/Spanish/strings.po [hunk 1]
```diff
@@ -14,7 +15,7 @@ msgstr ""
 "Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
 "POT-Creation-Date: 2012-07-03 05:24+0000\n"
 "PO-Revision-Date: 2013-04-29 08:35+0000\n"
-"Last-Translator: giftie <giftie61@hotmail.com>\n"
+"Last-Translator: giftie <roliverosc@hotmail.com>\n"
 "Language-Team: Spanish (Spain) (http://www.transifex.com/projects/p/cdART_Manager_XBMC_ADDON/language/es_ES/)\n"
```
Comment (stefansielaff) [inline resources/language/Spanish/strings.po]: Thanks for the awesome work! Just one little favour, could you update the line to your name here?

**Review by stefansielaff — CHANGES_REQUESTED** [review]

File: resources/language/Spanish/strings.po (hunk 1 above)
Comment (roliverosc-zz) [reply resources/language/Spanish/strings.po]: Fix regards.  Sorry I had not seen it.

**Review by stefansielaff — APPROVED** [review]

**Comment by stefansielaff** [comment]
Awesome, thanks :)

**Comment by roliverosc-zz** [comment]
Thank you. Regards

**Pull request merged**